simulator/obj_canvas.py

```diff
@@ -307,8 +274,54 @@ def check_collision(cls, obj1, obj2, threshold=0):
       collision[1] = True
 
     return collision
+    
+class Shape(CanvasObject):
+  """ Class that adds position functionality to CanvasObject"""
+  
+  def __init__(self, canvas, pos, **kwargs):
+    """
+    Args:
+      canvas: The canvas to draw on.
+      pos: The center position of the shape."""
```

Comment:
I would add a note in the docstring about all keyword arguments being passed transparently to the superclass constructor.